src/gt4py/next/program_processors/runners/dace/transformations/loop_blocking.py

```diff
@@ -380,6 +380,14 @@ def _classify_node(
             ):
                 return False
 
+        elif isinstance(node_to_classify, dace.nodes.NestedSDFG):
+            # Similar checks apply to a nested SDFG node
+            if not all(
+                isinstance(out_edge.dst, dace_nodes.AccessNode)
+                for out_edge in state.out_edges(node_to_classify)
```

Comment:
```suggestion
                for out_edge in state.out_edges(node_to_classify)
                if not out_edge.data.is_empty()
```

In addition you have to inspect if the nested SDFG uses the blocking parameter.
I am not sure if checking `free_symbols` is enough.
Consider the following (partial) symbol mappings; where `k` is the block parameter:
- `{'k': 'k'}` in this case it is certainly enough to check `free_symbol`.
- `{'inner_value': 'k'}` as far as I understand `k` will not be a free symbol of the nested SDFG.

So I think to be fully sure you will have to do something along:
```python
any('k' in ss for ss in [symbol_mapping.keys(), symbol_mapping.values(), nsdfg.free_symbol])
```
Maybe the check on free symbol is overkill.
Also be aware the `symbol_mapping.keys()` returns strings while `symbol_mapping.values()` returns symbols (they imposed some type casting in their properties but I am not sure how good it is).
Also be aware of cases such as `k + 10` or `k_othersymbol` which makes matching harder.